brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/brevproduksjon/bestiller/DokgenBrevproduksjonTjeneste.java

```diff
@@ -143,11 +146,11 @@ private void distribuerBrevOgLagHistorikk(DokumentHendelse dokumentHendelse, Dok
                         innsynMedVedlegg,
                         dokumentHendelse.getBehandlingUuid(),
                         saksnummer,
-                        dokumentHendelse.getBestillingUuid()));
+                        UUID.randomUUID())); //ToDo ANJA fjerne når vi ikke lenger krever Uuid på historikkinnslag
```

Comment:
Har du tilgang til Dokdist dokumentasjon av denne tjenesten? Må batchId være unikt også?

Reply:
Ellers ville jeg sendt samme string som ved oppretting av journalposten.

Reply:
Det er ikke krav til at batch id må være unik - men jeg tenker det er å følge samme mønster, da blir det konsekvent

Reply:
Det høres lurt ut. Go for it! :)